Userland/Libraries/LibGfx/Path.cpp

```diff
@@ -417,6 +417,12 @@ Path Path::copy_transformed(Gfx::AffineTransform const& transform) const
     return result;
 }
 
+void Path::transform(AffineTransform const& transform)
+{
+    for (auto& point : m_points)
```

Comment:
Isn't changing in-place and caching paths a somewhat dangerous combination?

Reply:
The cache returns a `Path const&`, so to use `.transform()` on the path you'd have to do an evil `const_cast`.